### ISSUE TITLE
Implement training process

### ISSUE THREAD

**Issue by LuXoN1999** [LuXoN1999/Segmentation-of-abdominal-organs-from-medical-images#15]
Implement loop for training the model. Define logic for training and validation phase.

# TODO

- create function which starts the training process
- log loss scores
- display loss scores while training with TensorBoard
- ADDITIONAL:
  - add all neccessary logic like optimizer, LR scheduler definition and hyperparameters, generating dataloaders etc.
  - since the loss calculation functions were not implemented correctly, move them to the .ipynb file for model training

# DOCUMENTATION

https://pypi.org/project/tensorboard/
